.github/workflows/check-generated-values.yml

```diff
@@ -0,0 +1,58 @@
+on: push
+name: Check values files
+jobs:
+  diff:
+    runs-on: ubuntu-latest
+    env:
+      TMP_DIR: "/tmp/shared"
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - name: Create directories
+        run: |
+          mkdir -p ~/.local/bin
+          mkdir -p "${TMP_DIR}"
+          chmod 777 "${TMP_DIR}"
+      - name: Create helmfile docker shim
```

Comment:
Is it a good idea to use helmfile and yq via docker here? Seems expensive and overly complex, but maybe it's fine. At least it keeps using the latest version in this way

Reply:
Seems "fine" to me; it may be smarter to have built a specific image or action for this but so far I don't totally hate it